Duplicati/Library/Backend/TahoeLAFS/TahoeBackend.cs

```diff
@@ -19,287 +19,243 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
 // DEALINGS IN THE SOFTWARE.
 
+using System.Net;
+using System.Reflection;
+using System.Runtime.CompilerServices;
 using Duplicati.Library.Common.IO;
 using Duplicati.Library.Interface;
 using Duplicati.Library.Utility;
 using Duplicati.Library.Utility.Options;
 using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-using System.Net;
-using System.Runtime.CompilerServices;
+using Uri = Duplicati.Library.Utility.Uri;
+
+namespace Duplicati.Library.Backend;
 
-namespace Duplicati.Library.Backend
+public class TahoeBackend : IStreamingBackend
 {
-    public class TahoeBackend : IBackend, IStreamingBackend
+    /// <summary>
+    /// Base URL for the Tahoe-LAFS backend
+    /// </summary>
+    private readonly string? _url;
+        
+    /// <summary>
+    /// The timeout options for API requests.
+    /// </summary>
+    private readonly TimeoutOptionsHelper.Timeouts? _timeouts ;

```

Comment:
Instead of making it nullable, you can make the default constructor with:
```
{
_timeouts = null!;
_certificateOptions = null!;
}
```

That way we avoid using `!` overrides in the rest of the code, and once we get around to removing the default constructor, everything works without `!` overrides.